src/main/java/org/sonar/plugins/findbugs/FindbugsConfiguration.java

```diff
@@ -57,16 +68,14 @@
 
   private final FileSystem fileSystem;
   private final Configuration config;
-  private final RulesProfile profile;
-  private final FindbugsProfileExporter exporter;
+  private final ActiveRules activeRules;
   private final JavaResourceLocator javaResourceLocator;
 
-  public FindbugsConfiguration(FileSystem fileSystem, Configuration config, RulesProfile profile, FindbugsProfileExporter exporter,
```

Comment:
This change seems to break the plugin.

```
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 7.781 s
[INFO] Finished at: 2019-01-28T18:39:50-05:00
[INFO] Final Memory: 19M/337M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.sonarsource.scanner.maven:sonar-maven-plugin:3.3.0.603:sonar (default-cli) on project sonar-java-only: org.sonar.plugins.findbugs.FindbugsConfiguration has unsatisfied dependency 'class org.sonar.api.profiles.RulesProfile' for constructor 'public org.sonar.plugins.findbugs.FindbugsConfiguration(org.sonar.api.batch.fs.FileSystem,org.sonar.api.config.Configuration,org.sonar.api.profiles.RulesProfile,org.sonar.plugins.findbugs.FindbugsProfileExporter,org.sonar.plugins.java.api.JavaResourceLocator)' from org.sonar.core.platform.ComponentContainer$ExtendedDefaultPicoContainer@7b2ccba5:110<[Immutable]:org.sonar.core.platform.ComponentContainer$ExtendedDefaultPicoContainer@7a0f244f:463<[Immutable]:org.sonar.core.platform.ComponentContainer$ExtendedDefaultPicoContainer@2a47597:52<| -> [Help 1]
```

Reply:
In fact, the last commit on this PR is fixing this very issue.